app/src/main/java/com/duckduckgo/app/notification/model/WebsiteNotification.kt

```diff
@@ -73,3 +78,51 @@ open class WebsiteNotificationSpecification(
         const val WEBSITE_KEY = "websiteKey"
     }
 }
+
+// This is not used at the moment.
+// @ContributesMultibinding(AppScope::class)
```

Comment:
Commented this out as it's not used. Added it as I made changes to the notifications and didn't want to lose the behaviour of this one in case it's ever needed.

Reply:
Removed :+1: